connectors/camel-sjms2-kafka-connector/pom.xml

```diff
@@ -48,7 +48,10 @@
     <dependency>
       <groupId>org.apache.activemq</groupId>
       <artifactId>activemq-client</artifactId>
-      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.activemq</groupId>
+      <artifactId>artemis-jms-client</artifactId>
```

Comment:
Is this client capable of all protocols that `activemq-client` support? In other words can we use just this dependences and get rid of `activemq-client`?

Reply:
Nope, unfortunately only Artemis CORE. To support OpenWire, AMQP and CORE we need three distinct dependencies. Defaulting to OpenWire (activemq-client) is good because the connector works with both ActiveMQ and Artemis.

Reply:
Got you, thanks!